pyproject.toml

```diff
@@ -47,6 +47,7 @@ dependencies = [
     "elasticsearch[async]==8.6.1",
     "elastic-transport==8.4.1",
     "urllib3==1.26.18",
+    "aiohttp==3.8.6",
```

Comment:
Would you mind adding the link to the aiohttp issue too? That way we'll know when we can upgrade